wrapspawner/wrapspawner.py

```diff
@@ -210,5 +215,65 @@ def clear_state(self):
         super().clear_state()
         self.child_profile = ''
 
+class DockerProfilesSpawner(ProfilesSpawner):
+    default_profiles = List(
```

Comment:
It would be nice to add a brief docstring for the class perhaps documenting what values/settings can be passed in the default profiles.